builtin/credential/github/path_login.go

```diff
@@ -68,38 +69,36 @@ func (b *backend) pathLogin(ctx context.Context, req *logical.Request, data *fra
 		return nil, err
 	}
 
-	resp := &logical.Response{
-		Auth: &logical.Auth{
-			InternalData: map[string]interface{}{
-				"token": token,
-			},
-			Policies: verifyResp.Policies,
-			Metadata: map[string]string{
-				"username": *verifyResp.User.Login,
-				"org":      *verifyResp.Org.Login,
-			},
-			DisplayName: *verifyResp.User.Login,
-			LeaseOptions: logical.LeaseOptions{
-				TTL:       config.TTL,
-				MaxTTL:    config.MaxTTL,
-				Renewable: true,
-			},
-			Alias: &logical.Alias{
-				Name: *verifyResp.User.Login,
-			},
+	auth := &logical.Auth{
+		InternalData: map[string]interface{}{
+			"token": token,
+		},
+		Policies: verifyResp.Policies,
```

Comment:
I don't think this is necessary here since we are overwriting it below?

Reply:
Yep, will remove.